src/main/java/org/candlepin/resource/CdnResource.java

```diff
@@ -44,18 +46,25 @@ public class CdnResource implements CdnApi {
     private CdnCurator curator;
     private CdnManager cdnManager;
     private ModelTranslator translator;
+    private PagingUtilFactory pagingUtilFactory;
```

Comment:
Another win from Sonarqube (wow!). This is unused after the removal of paging from the CDN accessor.